tests/add_subdirectory/client.cpp

```diff
@@ -1,3 +1,3 @@
 #include <vulkan/layer/vk_layer_settings.h>
 
-bool foobar() { return vku::IsLayerSetting("layer_name", "setting_key"); }
\ No newline at end of file
+bool foobar() { return vlHasLayerSetting("setting_key"); }
```

Comment:
This is bike shedding but should the prefix be `vl`?

Just as other options.

`vul` for `vulkan utility libraries`

`vulkanLayerSettings` for verbosity.

~~vk~~ I think should be off the table since it's not part of the official vulkan headers.

vl also makes sense for `vulkan layer` though. So I don't really have a strong opinion to be honest.

Reply:
That remains an open question to me too! And I was wondering what you where planning for the validation layer libraries so we could create a consistent approach for all this.

Reply:
So using a single namespace for everything?

Reply:
I don't have too much frame of reference for a package that exports multiple libraries. I've never really dealt with Boost or Qt so I don't know what the expected standard is.

Reply:
I think for now the namespace should just be `Vl`. When another library gets added we will see.

Reply:
Agreed!